utils/exporters/blender/addons/io_three/exporter/api/object.py

```diff
@@ -395,7 +405,12 @@ def extract_mesh(obj, options, recalculate=False):
     mesh_node.calc_normals()
     mesh_node.calc_tessface()
     scale_ = options.get(constants.SCALE, 1)
-    mesh_node.transform(mathutils.Matrix.Scale(scale_, 4))
+    global_matrix = (mathutils.Matrix.Scale(scale_, 4) *
+                     axis_conversion(to_forward='-Z',
+                                     to_up='Y',
+                                     ).to_4x4())
+
```

Comment:
It is not necessary to calculate the `global_matrix` for every mesh. Any suggestions where to put it instead?